subsys/shell/shell.c

```diff
@@ -765,7 +765,7 @@ static void shell_state_collect(const struct shell *shell)
 					cursor_next_line_move(shell);
 				} else {
 					/* Command execution */
-					shell_execute(shell);
+					(void)shell_execute(shell);
```

Comment:
you could add at least log message here if shell execution fails.

Reply:
I would like to avoid that, in my opinion logging failed execution should be implemented in command handler.

Reply:
ok